cves/kernel/CVE-2022-1671.yml

```diff
@@ -55,7 +55,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A vulnerability was discovered in the Linux kernel's code,
+  specifically in a function called rxrpc_preparse_s in the file
+  net/rxrpc/server_key.c. This flaw could be exploited by a local attacker to
+  either crash the system or reveal sensitive internal kernel information.
```

Comment:
This is just a little specific with the description. Could you go into more detail about maybe what that function does or like the general purpose of that area of the code? Like why is it able to reveal that kind of information? what does it do?